components/filter/filter-tags.js

```diff
@@ -0,0 +1,142 @@
+import '../colors/colors.js';
+import '../tag-list/tag-list.js';
+import '../tag-list/tag-list-item.js';
+import { css, html, LitElement } from 'lit';
+import { bodyCompactStyles } from '../typography/styles.js';
+import { IdSubscriberController } from '../../controllers/subscriber/subscriberControllers.js';
+import { LocalizeCoreElement } from '../../helpers/localize-core-element.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+/**
+ * A tag-list allowing the user to see (and remove) the currently applied filters.
+ */
+class FilterTags extends RtlMixin(LocalizeCoreElement(LitElement)) {
+	static get properties() {
+		return {
+			/**
+			 * REQUIRED: Id(s) (space-delimited) of the filter component(s) to subscribe to
+			 * @type {string}
+			 */
+			filterIds: { type: String, attribute: 'filter-ids' },
+			/**
+			 * The text displayed in this component's label
+			 * @default "Applied Filters:"
+			 * @type {string}
+			 */
+			labelText: { type: String, attribute: 'label-text' }
+		};
+	}
+
+	static get styles() {
+		return [bodyCompactStyles, css`
+			:host {
+				display: inline-block;
+				width: 100%;
+			}
+			:host([hidden]) {
+				display: none;
+			}
+
+			.d2l-filter-tags-wrapper {
+				display: flex;
+			}
+			d2l-tag-list {
+				flex: 1;
+			}
+
+			.d2l-filter-tags-label {
+				display: inline-block;
+				font-weight: bold;
+				margin-right: 0.25rem;
+				padding-top: 0.15rem;
+			}
+			:host([dir="rtl"]) .d2l-filter-tags-label {
+				margin-left: 0.25rem;
+				margin-right: 0;
+			}
+
+			.d2l-filter-tags-none-label {
+				color: var(--d2l-color-corundum);
+				display: inline-block;
+				font-style: italic;
+				padding-top: 0.15rem;
+			}
+		`];
+	}
+
+	constructor() {
+		super();
+		this.labelText = '';
+
+		this._allActiveFilters = new Map();
+		this._filters = new IdSubscriberController(this,
+			{ onUnsubscribe: this._removeLostFilter.bind(this) },
+			{ idPropertyName: 'filterIds' }
+		);
+	}
+
+	render() {
+		let numActiveFilters = 0;
+		const allActiveFilters = Array.from(this._allActiveFilters);
+		const labelText = this.labelText || this.localize('components.filter.appliedFilters');
```

Comment:
Minor: I wonder if we make this a tad clearer by just doing this directly in the `d2l-tag-list`'s `description`, and then just use `this.labelText` down in https://github.com/BrightspaceUI/core/pull/2488/files#diff-a2bd69c4105bdfbd42a980a991c5a164adb7d280fc40947cef82104be6349867R109.  Make it obvious we always have something for `d2l-tag-list`, but not necessarily a visible label.

Reply:
Yup that would actually probably make more sense